src/widget/wtracktableview.cpp

```diff
@@ -812,12 +811,11 @@ void WTrackTableView::setSelectedTracks(const QList<TrackId>& trackIds) {
     }
 
     for (const auto& trackId : trackIds) {
-        const QLinkedList<int> gts = pTrackModel->getTrackRows(trackId);
+        const std::list<int> gts = pTrackModel->getTrackRows(trackId);
```

Comment:
We don't care about the actual container type, it only needs to support iteration with a range-based for loop. I would propose to use `auto` here to avoid any explicit dependencies on the actual type. Otherwise we need to modify this line again when deciding to change the type from std::list to QVector.

@daschuer Do you agree?